README.md

```diff
@@ -114,21 +129,51 @@ Create a `secretproviderclasses` resource to provide provider-specific parameter
     | subscriptionId         | no      | [__*required for version < 0.0.4*__] subscription ID containing key vault instance                   | ""            |
     | tenantId               | yes      | tenant ID containing key vault instance                         | ""            |
 
-1. Update your [linux deployment yaml](examples/nginx-pod-secrets-store-inline-volume-secretproviderclass.yaml) or [windows deployment yaml](examples/windows-pod-secrets-store-inline-volume-secretproviderclass.yaml) to use the Secrets Store CSI driver and reference the `secretProviderClass` resource created in the previous step
+1. Update your [linux deployment yaml](examples/nginx-pod-secrets-store-inline-volume-secretproviderclass.yaml) or [windows deployment yaml](examples/windows-pod-secrets-store-inline-volume-secretproviderclass.yaml) to use the Secrets Store CSI driver and reference the `secretProviderClass` resource created in the previous step. 
 
-    ```yaml
-    volumes:
-      - name: secrets-store-inline
-        csi:
-          driver: secrets-store.csi.k8s.io
-          readOnly: true
-          volumeAttributes:
-            secretProviderClass: "azure-kvname"
+        If you did not change the name of the secretProviderClass previously, no changes are needed.
+    
+        ```yaml
+        volumes:
+          - name: secrets-store-inline
+            csi:
+              driver: secrets-store.csi.k8s.io
+              readOnly: true
+              volumeAttributes:
+                secretProviderClass: "azure-kvname"
+        ```
+
+  1. Select and complete an option from [below to enable access to the Key Vault](####Provide-Identity-to-Access-Key-Vault)
+
+  1. Deploy the secretProviderClass
+
+     `kubectl apply -f ./examples/v1alpha1_secretproviderclass.yaml`
+
+  1. Deploy the Linux deployment yaml
+
+     `kubectl apply -f ./examples/nginx-pod-secrets-store-inline-volume-secretproviderclass.yaml`
+
+#### Validate the secret
```

Comment:
we can perhaps consider using the same stuff content from https://github.com/kubernetes-sigs/secrets-store-csi-driver#secret-content-is-mounted-on-pod-start

Reply:
@jluk We should probably copy this part of the content from the link @ritazh posted -

To validate, once the pod is started, you should see the new mounted content at the volume path specified in your deployment yaml.
```
kubectl exec -it nginx-secrets-store-inline ls /mnt/secrets-store/
foo
```

I don't think we need a link to the driver readme (just so we can avoid the extra hop to different readme for reading about validation). WDYT?

Reply:
made another change per ask

Reply:
Thank you @jluk! Looks good to me.